misc/server_log.yml

```diff
@@ -37,6 +37,11 @@ loggers:
     - core_tornado_debug_log_handler
     level: DEBUG
     propagate: true
+  sqlalchemy.engine:
+    handlers:
+      - server_handler
```

Comment:
```suggestion
```


otherwise this will cause duplicate logs, as the server_handler handler is both attached at the root and here.


(applies everywhere)

Reply:
Morever, why would we need a custom config? We already set the root level?

Reply:
I confess that my understanding of custom logging is a bit limited.
By custom config do you mean just the handler part or simply adding this entry to the `loggers` ?